index.html

```diff
@@ -0,0 +1,533 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/5b1eeceadd.js" crossorigin="anonymous"></script>
+  <title>TNW-Website-Clone</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav class="nav">
+      <div class="nav-logo">
+        <a href="index.html"><img src="./images/tnw-red.svg" alt=""></a>
+        <a href=""><i class="fas fa-bars bar fa-2x"></i></a>
+      </div>
+      <div class="lists">
+        <div class="list-container">
+          <ul>
+            <li><a href="">News</a></li>
+            <li><a href="">Events</a></li>
+            <li><a href="">Business</a></li>
+            <li><a href="">AMAs</a></li>
+            <li><a href="">Spaces</a></li>
+            <li><a href="">Terms&Conditions</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Advertise</a></li>
+            <li><a href="">Jobs</a></li>
+            <li><a href="">Contact</a></li>
+          </ul>
+        </div>
+        <div class="nav-list3">
+          <ul class="nav-section">
+            <li><a href="">LATEST</a></li>
+            <li><a href="">HARD FORK</a></li>
+            <li><a href="">PLUGGED</a></li>
+            <li><a href="">README</a></li>
+            <li><a href="">GROWTH QUARTERS</a></li>
+            <li><a href="">SHIFT</a></li>
+            <li><input type="text" name="search" id="search"></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main class="main-container">
+    <div class="first-section">
+      <article class="flex-pictures">
+        <div class="first-img">
+          <img src="./images/tiktok-censorship-lgbtq-796x417.jpg" alt="" class="main-img img-left">
+          <div class="inside">
+            <h5>INSIGHT</h5>
+            <h2 class="inside-heading">TikTok says it’s committed to diversity — its history of censorship says
+              otherwise</h2>
+            <span class="small-text">by Bryan Clark</span>
+          </div>
+        </div>
+        <div class="second-img">
+          <div>
+            <img src="./images/hdc-2020-img-1.jpg" alt="" class="main-img img-right">
+          </div>
+          <div>
+            <img src="./images/remote-work-home-gq-368x276.png" alt="" class="main-img img-right">
+          </div>
+          <div class="inside">
+            <h5>TECH</h5>
+            <h2 class="black inside-heading">5 lessons I’ve learned working from home</h2>
+            <span class="black-small-text">by Tom Popomaronis 11 hours ago</span>
+          </div>
+        </div>
+      </article>
+    </div>
+    <section class="hard-fork">
+      <h2 class="latest-heading">Latest news</h2>
```

Comment:
- [ ] There is an empty space on the right side of the latest news section in the medium screen view. Please fix it to create the same UI with the original page.

![image](https://user-images.githubusercontent.com/60184961/93029024-da4bab80-f620-11ea-9076-872c381923b5.png)